src/Compilers/CSharp/Test/Symbol/DeclarationTests.cs

```diff
@@ -293,7 +293,7 @@ public class B
 
             var countedTree = new CountedSyntaxTree(foreignType);
 
-            var compilation = CreateCompilation(new SyntaxTree[] { underlyingTree, countedTree }, skipUsesIsNullable: true);
+            var compilation = CreateCompilation(new SyntaxTree[] { underlyingTree, countedTree }, skipUsesIsNullable: true, options: TestOptions.ReleaseDll);
```

Comment:
>, options: TestOptions.ReleaseDll [](start = 122, length = 33)

Is this change necessary?

Reply:
Yes, it changes all the `AccessCounts` to 2 instead of 1.

---
In reply to: [554043500](https://github.com/dotnet/roslyn/pull/50325#discussion_r554043500) [](ancestors = 554043500)

Reply:
It feels suspicious that changing the output kind would have this effect. I think we need to understand what is going on.

---
In reply to: [554099386](https://github.com/dotnet/roslyn/pull/50325#discussion_r554099386) [](ancestors = 554099386,554043500)

Reply:
Ah, ok. It's because calls to `GetRoot()` are counted, so the new check for "does this syntax have top-level statements" increments the counter. I'll add a remark to `CountedSyntaxTree` about it.

---
In reply to: [554109951](https://github.com/dotnet/roslyn/pull/50325#discussion_r554109951) [](ancestors = 554109951,554099386,554043500)